src/applications/personalization/dashboard-2/tests/unit/healthcare.unit.spec.jsx

```diff
@@ -10,126 +10,87 @@ describe('HealthCare component', () => {
   let view;
   let initialState;
 
-  context('when user has the `messaging` service', () => {
-    beforeEach(() => {
-      window.VetsGov = { pollTimeout: 1 };
-      mockFetch();
-      initialState = {
-        user: {
-          profile: {
-            services: ['messaging'],
-          },
+  beforeEach(() => {
```

Comment:
Updated these tests, because we want to always fetch the messaging data and always show the messaging section - regardless of whether 'messaging' is contained within the services array.